libs/application/templates/driving-license/src/dataProviders/QualityPhotoProvider.ts

```diff
@@ -35,7 +35,7 @@ export class QualityPhotoProvider extends BasicDataProvider {
 
     return {
       success: response.data.qualityPhoto.success,
-      qualityPhoto: response.data.qualityPhoto.qualityPhoto,
+      qualityPhoto: response.data.qualityPhoto?.qualityPhoto,
```

Comment:
Shouldn't the optional chaning operator also be used in L#37 and have default false on success if `qualityPhoto` can be `undefined` or `null`?